cobalt/worker/service_worker_consts.cc

```diff
@@ -45,5 +45,23 @@ const char ServiceWorkerConsts::kServiceWorkerAllowed[] =
 
 const char ServiceWorkerConsts::kSettingsJson[] =
     "service_worker_settings.json";
+
```

Comment:
This looks like a change that already is in https://github.com/youtube/cobalt/pull/173/files